src/resolve/resolved-dns-transaction.c

```diff
@@ -1350,7 +1350,16 @@ static int dns_transaction_prepare(DnsTransaction *t, usec_t ts) {
         }
 
         if (t->n_attempts >= TRANSACTION_ATTEMPTS_MAX(t->scope->protocol)) {
-                dns_transaction_complete(t, DNS_TRANSACTION_ATTEMPTS_MAX_REACHED);
+                DnsTransactionState result;
+
+                if (t->scope->protocol == DNS_PROTOCOL_LLMNR)
+                        /* If we didn't find anything on LLMNR, it's not an error, but a failure to resolve
+                         * the name. */
+                        result = DNS_TRANSACTION_NOT_FOUND;
+                else
+                        result = DNS_TRANSACTION_ATTEMPTS_MAX_REACHED;
```

Comment:
hmm, interesting change. so if we did this for mdns instead of llmnr (which works quite similar in most ways) this would break stuff, since the probing logic waits for ATTEMPTS_MAX_REACHED to be hit before establishing the name. LLMNR works differently though, hence this is probably OK.

Hence, let's merge, i really hope i didn't miss the impact of this though.